### PR TITLE
Update submodule wxWidgets

### PR DESCRIPTION
**Pull request by Zangetsu38**
Fix Serval warning on VS 2015 and other commit https://github.com/wxWidgets/wxWidgets/commits/master

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,8 +1,6 @@
 [submodule "wxWidgets"]
 	path = wxWidgets
 	url = https://github.com/wxWidgets/wxWidgets
-	ignore = dirty
```
Comment (Nekotekina) [inline .gitmodules]: Why removed?

Comment (Zangetsu38) [inline .gitmodules]: Because this line is for a while and it is useless.

Comment (Bigpet) [inline .gitmodules]: I can tell you what it's for. Their `.gitignore` does not contain all the various Visual Studio temporary build files. So if you build it there's some additional files that git considers for staging, this makes sure that changing revisions won't be unsuccessful when there's unexpected files and that it doesn't tag the submodule as "-dirty" just for the few additional files (doesn't show the submodule in the staging area).

**Comment by Bigpet** [comment]
We don't need to update the revision of wxWidgets each week.

File: .gitmodules (hunk 1 above)
Comment (Zangetsu38) [inline .gitmodules]: I did not know at that, I did not think the ".gitmodule" could be used to that, I thought it was the ".gitignore" who spanked it.
But I do not understand, I deleted it, and when I actually delivered after compiling it does not appear to wxWdigets my file, so it should not be used in the end.

**Comment by Zangetsu38** [comment]
@Bigpet I thought if for once, even if it is true that it is recent, the change was significant for VS 2015 I think.

**Comment by danilaml** [comment]
@Zangetsu38 current revision already contains vs15 changes.

**Comment by Zangetsu38** [comment]
@danilaml euu ? I Know, i speak other change for VS 2015 eg : https://github.com/wxWidgets/wxWidgets/commit/36e31e515e9016db7327a0fb801571bef8b01b9f

**Comment by danilaml** [comment]
@Zangetsu38 just dome warnings silenced. Not that important. Could wait for something more functional.

**Pull request closed without merging**